R/newarima2.R

```diff
@@ -711,10 +713,12 @@ myarima <- function(x, order = c(0, 0, 0), seasonal = c(0, 0, 0), constant=TRUE,
   m <- frequency(x)
   use.season <- (sum(seasonal) > 0) & m > 0
   diffs <- order[2] + seasonal[2]
-  if (approximation) {
-    method <- "CSS"
-  } else {
-    method <- "CSS-ML"
+  if(is.null(method)){
```

Comment:
The default setting will have method="CSS-ML", so this will not be triggered.

Reply:
Great, I thought there was something I had missed!